tests/acl/test_acl.py

```diff
@@ -349,7 +349,7 @@ def teardown_rules(self, dut):
         dut.command("config acl update full {}".format(remove_rules_dut_path))
 
     @pytest.fixture(scope="class", autouse=True)
-    def acl_rules(self, duthost, localhost, setup, acl_table, populate_vlan_arp_entries):
+    def acl_rules(self, duthosts, rand_one_dut_hostname, localhost, setup, acl_table, populate_vlan_arp_entries):
```

Comment:
There is one more miss in this file. I fixed it in PR #2534. So that means the method you are using to identify these change still have issue.